### DISCUSSION TITLE
Where does pyload store its tmp files?

### DISCUSSION THREAD

**Discussion by brianjessup** [pyload/pyload#4000] [Help]
It seems like during download pyload stores the partial files in a tmp directory and then transfers them over to the specified download folder. Is that correct?

What is the path to that folder?

Is there a way to alter where the tmp files are stored?

**Comment by GammaC0de** [comment]
No, pyload split the download into chunks and downloads the chunks to download directory with file extension .chunk0, chunk1 and so on.
